keyboards/handwired/polykybd/lang/_set_cell.py

```
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+"""Set a single cell in the key_lut sheet (sheet2.xml) of lang_lut.xlsx, in place.
+Inserts the <c> in column order if absent, or replaces it if present. Only
+sheet2.xml is rewritten; every other zip entry (and the formula caches) is
+copied byte-for-byte.
+
+usage: _set_cell.py <xlsx> <row> <col1based> <value> [num|str]
+"""
+import sys, re, zipfile, shutil, os
+from xml.sax.saxutils import escape
+
+XLSX=sys.argv[1]; ROW=int(sys.argv[2]); COL=int(sys.argv[3]); VAL=sys.argv[4]
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)